api/teams/tui.py

```diff
@@ -140,16 +140,23 @@ def change_tech_leader(team):
 
 
 def add_members(team, turma):
+    limitation = False
     while True:
         should_add = input("Deseja adicionar mais um membro (S/N)? ")
         if should_add == "S" or should_add == "s":
             print("Selecione um Membro")
             new_member = search_and_select_student(turma, excludes=team["members"])
+            limitation = student_limitation(new_member, turma)
```

Comment:
antes de chamar o `student_limitation()`, tem que fazer o `if new_member is None:` ou então a função vai receber um `None` no parâmetro `student` e depois vai quebrar no `if student["id"] == member["id"]:` (linha 99 do `api/teams/repository.py`)